content/panel_release_1.3.ipynb

```diff
@@ -0,0 +1,1970 @@
+{
```

Comment:
<p>Didn't realize there was in line commenting:</p><p><a href="https://app.reviewnb.com/holoviz-dev/blog/pull/26/discussion" target="_blank">https://app.reviewnb.com/holoviz-dev/blog/pull/26/discussion/</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/holoviz-dev/blog/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>